### PR TITLE
Create Sitemap Entity

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
**Description**
This PR with a test that creates Sitemap

**Steps:**

1. Log in as admin user from data set.
2. Navigate to Marketing > SEO and Search > Site Map.
3. Click "Add Sitemap" button.
4. Fill out all data according to data set.
5. Click "Save" button.
6. Perform all assertions:
**Fixed Issues:**
#338 Convert CreateSitemapEntityTest to MFTF

### CODE REVIEW

File: app/code/Magento/Sitemap/Test/Mftf/ActionGroup/AssertAdminSitemapMessageActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminSitemapMessageActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Sitemap/Test/Mftf/ActionGroup/AssertAdminSitemapMessageActionGroup.xml]: I just want to doublecheck that we actually need this action group and related selectors. It does look like that we may use the [following](https://github.com/magento/magento2/blob/83e3487a12e1165dde0c9175fbad45b9bfe018c3/app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertMessageInAdminPanelActionGroup.xml#L11) existing action group. As far as I see, it uses the same selectors.

File: dev/tests/functional/tests/app/Magento/Sitemap/Test/TestCase/CreateSitemapEntityTest.xml [hunk 2]
```diff
@@ -8,20 +8,20 @@
...
         <variation name="CreateSitemapEntityTestVariation2">
-            <data name="tag" xsi:type="string">severity:S3</data>
+            <data name="tag" xsi:type="string">severity:S3,mftf_migrated:yes</data>
             <data name="sitemap/data/sitemap_filename" xsi:type="string">%isolation%</data>
             <data name="sitemap/data/sitemap_path" xsi:type="string">/</data>
             <constraint name="Magento\Sitemap\Test\Constraint\AssertSitemapFailPathSaveMessage" />
         </variation>
         <variation name="CreateSitemapEntityTestVariation3">
-            <data name="tag" xsi:type="string">severity:S3</data>
+            <data name="tag" xsi:type="string">severity:S3,,mftf_migrated:yes</data>
```
Comment (rogyar) [inline dev/tests/functional/tests/app/Magento/Sitemap/Test/TestCase/CreateSitemapEntityTest.xml]: One comma should be enough in this case 😉

```suggestion
            <data name="tag" xsi:type="string">severity:S3,mftf_migrated:yes</data>
```

File: app/code/Magento/Sitemap/Test/Mftf/ActionGroup/AssertAdminSitemapMessageActionGroup.xml (hunk 1 above)
Comment (DmitryTsymbal) [reply app/code/Magento/Sitemap/Test/Mftf/ActionGroup/AssertAdminSitemapMessageActionGroup.xml]: Indeed. Reused in all tests in the scope of this PR.
Thank you

File: dev/tests/functional/tests/app/Magento/Sitemap/Test/TestCase/CreateSitemapEntityTest.xml (hunk 2 above)
Comment (DmitryTsymbal) [reply dev/tests/functional/tests/app/Magento/Sitemap/Test/TestCase/CreateSitemapEntityTest.xml]: Such stupid inaccuracy from my side 😄 
Thank you for noticing!

File: app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingCreateSitemapEntityTest.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+    <!--
+     /**
+      * Copyright © Magento, Inc. All rights reserved.
+      * See COPYING.txt for license details.
+      */
+    -->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (rogyar) [inline app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingCreateSitemapEntityTest.xml]: Hi @DmitryTsymbal. Thank you for your changes. 
I've noticed that we have a minor issue in this test. During tests execution, we need to eliminate side-effects after every test so we have a "clean" state of the system. In the current test, we create a sitemap but don't remove it somehow. We need to remove it either in `<after>` section (preferable) or in the test body. I believe we may use the [following](https://github.com/magento/magento2/blob/5465fedb59f4b3120f6aea086cdeebb8cf28266d/app/code/Magento/Sitemap/Test/Mftf/ActionGroup/AdminMarketingSiteDeleteByNameActionGroup.xml#L11) action group for this purpose.

Comment (DmitryTsymbal) [reply app/code/Magento/Sitemap/Test/Mftf/Test/AdminMarketingCreateSitemapEntityTest.xml]: Hi @rogyar . Thank you for noticing! I have added mentioned Action Group for removing created sitemap into <after> section.
Please check latest commit.
Thank you!

**Pull request merged**